### DISCUSSION TITLE
[issue] Unable to get notification from my selfhosted ntfy instance

### DISCUSSION THREAD

**Discussion by Greatz08** [zen-browser/desktop#1183] [General]
i have selfhosted my ow ntfy instance and i am unable to receive the push notification in zen browser . Please give users the option to allow or block things.Too many unnecessary restrictions wont enhance privacy of users so please give users the options to enable and disable things like floorp does. It give user option to enable or disable and give brief text about the functionality of that option 

![image](https://github.com/user-attachments/assets/eb084519-f36c-4edb-af74-ecd2388e6528)

**Comment by emonadeo** [comment]
Privacy & Security → Permissions → Notifications → Settings

You can either directly add URLs there or uncheck „Block new requests asking to allow notifications“, which is enabled by default.
